services/QuillCMS/spec/models/response_spec.rb

```diff
@@ -41,4 +44,58 @@
       expect(new_response.valid?).to be true
     end
   end
+
+  describe '#wipe_question_cache' do
+    let(:question_uid) { 'some-unique-uid' }
+    let(:cache_key) { Response.questions_cache_key(question_uid) }
+
+    before do
+      Rails.cache.write(cache_key, 'cached content')
+    end
+
+    it 'clears the cache for the given question UID' do
+      expect(Rails.cache.read(cache_key)).to eq('cached content')
+
+      response = Response.create(question_uid: question_uid)
+      response.wipe_question_cache
+
+      expect(Rails.cache.read(cache_key)).to be_nil
```

Comment:
This is pure preference, so feel free to skip it, but I find this kind of logic more readable in an `expect to change` wrapper:
```ruby
expect do
  response = Response.create(question_uid: question_uid)
  response.wipe_question_cache
end.to change(Rails.cache.read(cache_key)).from('cached content').to(nil)
```